cmd/client/cmd/check.go

```diff
@@ -77,13 +90,18 @@ func check(rootArgs *rootArgs, printf, fatalf shared.FormatFn) {
 			fatalf("Failed to receive a response from Check RPC: %v", err)
 		}
 
-		printf("Check RPC returned %s", decodeStatus(response.Result))
-		dumpAttributes(printf, fatalf, response.AttributeUpdate)
-	}
+		<-sem
 
+		// To limit verbosity, only show responses for small repeat counts (otherwise logging slows down the test)
+		if rootArgs.repeat < 10 {
+			printf("Check RPC returned %s", decodeStatus(response.Result))
+			dumpAttributes(printf, fatalf, response.AttributeUpdate)
+		}
+	}
 	if err = stream.CloseSend(); err != nil {
 		fatalf("Failed to close gRPC stream: %v", err)
 	}
 
+	println("Done in ", time.Since(t), " avg(ms)=", (time.Since(t).Nanoseconds()/1000)/int64(rootArgs.repeat))
```

Comment:
another choice is 
dur := time.Since(t)
println("Done in ", dur, " avg=", dur /time.Duration(rootArgs.repeat))
go just prints time with units